typescript/cli/src/utils/version-check.ts

```diff
@@ -0,0 +1,11 @@
+import latestVersion from 'latest-version';
+
+import { logRed } from '../../logger.js';
+import { VERSION } from '../version.js';
+
+export async function checkVersion() {
+  const currentVersion = await latestVersion('@hyperlane-xyz/cli');
+  if (VERSION < currentVersion) {
+    logRed(`Please update your CLI to latest version(${currentVersion})`);
```

Comment:
Let's soften this since many CLI updates are minor patches or even empty updates (since hyperlane's packages all ship together). I suggest:
log(`Your CLI version: ${VERSION}, latest version: ${currentVersion}`);

Reply:
Sure, will change it